### ISSUE TITLE
Overly sensitive

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6054]
**Preferred term label:**
Overly sensitive

**Synonyms**
Highly sensitive
Hypersensitivity

**Definition (free text, please give PubMed ID)**
An abnormal emotion/affect behavior where individuals are highly sensitive to physical (via sound, sigh, touch, or smell) and or emotional stimuli and the tendency to be easily overwhelmed by too much information.

Source: https://www.additudemag.com/hypersensitivity-disorder-with-adhd/

**Parent term (use hpo.jax.org/app)**
Abnormal emotion/affect behavior

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ADHD

**Comment by pnrobinson** [comment]
I would use the label
Emotional hypersensitivity
Syn -- all of the above
def: Heightened emotional reactivity to environmental stimuli, including emotions of others.
PMID: 23250816
@nicolevasilevsky  -- thoughts?

**Comment by nicolevasilevsky** [comment]
that sounds great, thanks @pnrobinson.
I can assign myself and add this.
